starknet_py/net/schemas/gateway.py

```diff
@@ -299,15 +301,38 @@ def make_dataclass(self, data, **kwargs):
         return DeployedContract(**data)
 
 
+class DeclaredContractHashSchema(Schema):
+    class_hash = Felt(data_key="class_hash", required=True)
+    compiled_class_hash = Felt(data_key="compiled_class_hash", required=True)
+
+    @post_load
+    def make_dataclass(self, data, **kwargs) -> DeclaredContractHash:
+        return DeclaredContractHash(**data)
+
+
+class ReplacedClassSchema(Schema):
+    contract_address = Felt(data_key="address", required=True)
```

Comment:
```suggestion
    address = Felt(data_key="address", required=True)
```
Why add the contract_ prefix?

Reply:
It is named like that in the RPC specs. I can change it to just `address` if you think that is better?

Reply:
Hmm, it's a schema for gateway, where the field is `address` (as in data_key). Up to you.

Reply:
I'd prefer we follow RPC specs.